profile/georgia-reborn/scripts/Base/gr-main-components.js

```diff
@@ -4804,6 +4807,7 @@ class WaveformBar {
 			}
 			return false;
 		}
+		console.log(`File ${file} was successfully analyzed.`);
```

Comment:
We do not really need this, this can be removed. 
Only report when it failed to analyze, otherwise we know that analyze process was successful anyways.